app/components/file-menu.js

```diff
@@ -23,6 +23,12 @@ export default Ember.Component.extend({
     share() {
       prompt('Ctrl + C ;-)', window.location.href);
     },
+    embed() {
+      let src = window.location.href.split("?")[0];
+      src += "?numColumns=0";
+      let embedCode = `<iframe width="800" height="600" src="${src}"></iframe>`;
```

Comment:
Actually maybe I'm misunderstanding. According to this, using window.location.href is relatively safe, and generally more safe than alternatives: http://stackoverflow.com/questions/24078332/is-it-secure-to-use-window-location-href-directly-without-validation